### PR TITLE
test: Fix br_300_small_tables bug

### PR DESCRIPTION
**Pull request by zwj-coder**
### What problem does this PR solve? 
PR [#1285 ](https://github.com/pingcap/br/pull/1285) modified the br_300_small_tables case to extrace summary info from the br log;
```sh
restore_size=`tail -n 2 ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
```
This statement assume the summary info would always appear in last two lines of br log which is not the case.

### What is changed and how it works?
Change the statement above to 
```sh
restore_size=`cat ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'``
```

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note

 - No release note

### CODE REVIEW

File: tests/br_300_small_tables/run.sh [hunk 1]
```diff
@@ -83,11 +87,12 @@ done
...
 echo "restore 1/300 of the table start..."
 run_br restore table --db $DB  --table "sbtest100" --log-file $RESTORE_LOG -s "local://$TEST_DIR/$DB" --pd $PD_ADDR --no-schema
-restore_size=`tail -n 2 ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
+restore_size=`cat ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
 echo "restore data size is ${restore_size}"
 
-diff=$(calc "$backup_size-$restore_size*$TABLES_COUNT")
-echo ${diff}
+diff=$(calc "$backupv2_size-$restore_size*$TABLES_COUNT")
+diff=$(abs "$diff")
```
Comment (kennytm) [inline tests/br_300_small_tables/run.sh:94]: i don't think getting `abs`olute value is useful.

File: tests/br_300_small_tables/run.sh [hunk 2]
```diff
@@ -83,11 +82,11 @@ done
 rm -rf $RESTORE_LOG
 echo "restore 1/300 of the table start..."
 run_br restore table --db $DB  --table "sbtest100" --log-file $RESTORE_LOG -s "local://$TEST_DIR/$DB" --pd $PD_ADDR --no-schema
-restore_size=`tail -n 2 ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
+restore_size=`cat ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
```
Comment (kennytm) [inline tests/br_300_small_tables/run.sh:85]:
```suggestion
restore_size=`grep "restore data size" "${RESTORE_LOG}" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
```

**Review by kennytm — COMMENTED** [review]
rest LGTM

**Review by kennytm — APPROVED** [review]

**Review by Leavrth — APPROVED** [review]

**Pull request merged**